Day-4-5/src/components/SearchBar.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import TeamList from "../data/data.json";
+
+const SearchBar = ({ searchRestaurants }) => {
+  const [searchText, setSearchText] = useState("");
+
+  const filterRestaurants = (searchText) => {
```

Comment:
Please make this a utility function and make it pure. Then same function will be used anywhere else too. 
For example,
use TeamList in formSubmitHandler and pass it to filterRestaurants to make it a generic(reusable) function.

Reply:
You can add the same function to utils file too later.